### PR TITLE
let user specify exemptfee

### PR DESCRIPTION
**Pull request by ubbabeck**
# About PR

Implements #634 

### how it looks
set_exemptfee

### CODE REVIEW

**Comment by ademar111190** [comment]
@ubbabeck I don't see it on your screenshot. am I missing something?

**Comment by ubbabeck** [comment]
> @ubbabeck I don't see it on your screenshot. am I missing something?

I used the same text as we have in breezmobile, should we change it to `Exempt Fee in sats` instead ?

File: lib/bloc/payment_options/payment_options_bloc.dart [hunk 1]
```diff
@@ -56,10 +66,13 @@ class PaymentOptionsBloc extends Cubit<PaymentOptionsState> {
     _log.v("Fetching payments override settings");
     final enabled = await _preferences.getPaymentOptionsOverrideFeeEnabled();
     final proportional = await _preferences.getPaymentOptionsProportionalFee();
-    _log.v("Payments override fetched: enabled: $enabled, proportional: $proportional");
+    final excemptFeeMsat = await _preferences.getPaymentOptionsExemptFee();
+    _log.v(
+        "Payments override fetched: enabled: $enabled, proportional: $proportional, excemptFeeMsat: $excemptFeeMsat");
```
Comment (erdemyerebasmaz) [inline lib/bloc/payment_options/payment_options_bloc.dart:71]: Typo in `excemptFeeMsat`
should be `exemptFeeMsat`

Comment (ubbabeck) [reply lib/bloc/payment_options/payment_options_bloc.dart]: Fixed 😃

**Comment by ubbabeck** [comment]
## Updated translations 
exempt_fee

File: lib/bloc/payment_options/payment_options_bloc.dart (hunk 1 above)
Comment (erdemyerebasmaz) [reply lib/bloc/payment_options/payment_options_bloc.dart]: @ubbabeck there's one more thing remaining, the file name :)
`lib/routes/payment_options/widget/excempt_fee_widget.dart`

Comment (ubbabeck) [reply lib/bloc/payment_options/payment_options_bloc.dart]: For some reason this word tripped me up big time, believe all typos have been removed now. Except the remote branch name...

**Review by ademar111190 — APPROVED** [review]

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

**Pull request merged**